projects/ad7091r8-sdz/builds.json

```diff
@@ -0,0 +1,10 @@
+{
+    "maxim": {
+        "dummy_example_max78000_ad7091r8": {
```

Comment:
It looks like this project is tailored to run on max78000 and this means you're connecting the eval board to the max78000 with flywires. In this case it's not easy to figure out what the connections should be so I think it's necessary to add a documentation .rst file as well with a table showing the required HW connections.